upstream-builder.Dockerfile

```diff
@@ -13,8 +13,8 @@ FROM alpine:3
 
 COPY ["nsswitch.conf", "/etc/nsswitch.conf"]
 
-RUN chgrp -R 0 /etc && \
-    chmod -R g+rwx /etc
+#RUN chgrp -R 0 /etc && \
+#    chmod -R g+rwx /etc
```

Comment:
Should we revert these changes? These seem unrelated to the overall set of changes.

Reply:
They're required.  (I should've deleted the lines, grumble, grumble).  /etc is on a read-only filesystem, so these commands fail.